pkgs/development/libraries/tbb/default.nix

```diff
@@ -1,5 +1,8 @@
-{ stdenv, fetchurl }:
+{ stdenv, fetchurl, coreutils }:
```

Comment:
You shouldn't need to explicitly depend on `coreutils`, as it's part of the `stdenv` magic path. I think the main thing here is the `SHLIB_EXT`?